src/Application.php

```diff
@@ -268,10 +268,7 @@ public function initProfiler()
 
     public function initLocale()
     {
-        $configLocale = $this['config']->get('general/locale', Application::DEFAULT_LOCALE);
-        if (!is_array($configLocale)) {
-            $configLocale = [$configLocale];
-        }
+        $configLocale = (array) $this['config']->get('general/locale', Application::DEFAULT_LOCALE);
```

Comment:
`'foo'` will be cast to `['foo']`
`['foo']` will stay `['foo']`
`1` will be cast to `[1]`
`true` will be cast to `[true]`
…and so on

When I benchmarked it for 10,000,000 iterations the difference in performance was significant. :grin: